bower.json

```diff
@@ -1,11 +1,10 @@
 {
   "name": "adapt-contrib-confidenceSlider",
-  "version": "2.0.0",
+  "version": "2.1.1",
   "framework": "^2.0.0",
-  "homepage": "https://github.com/adaptlearning/adapt-contrib-confidenceSlider",
-  "issues": "https://github.com/adaptlearning/adapt-contrib-confidenceSlider/issues",
-  "repository": "git@github.com:adaptlearning/adapt-contrib-confidenceSlider.git",
-  "description": "A contributed two-part component that measures user confidence, using a sliding scale.",
+  "homepage": "https://github.com/chris-steele/adapt-contrib-confidenceSlider",
```

Comment:
Might want to purge your name from `homepage` and `issues` here. You know it'll just come back to bite you.